### PR TITLE
Gmsh write

### PR DESCRIPTION
**Pull request by zitkat**
Added read_data and write to Msh2MeshIO for partial support of GMSH 2.0 format, according to http://gmsh.info/doc/texinfo/gmsh.html#File-formats

### CODE REVIEW

**Comment by rc** [comment]
Thanks! Could you rebase on master so that the merge commits disappear? I can help with that, if needed.

File: sfepy/discrete/fem/meshio.py [hunk 1]
```diff
@@ -2822,7 +2902,7 @@ def read(self, mesh, omit_facets=True, **kwargs):
                 pass
 
             else:
-                output('skipping unknown entity: %s' % line)
+                # output('skipping unknown entity: %s' % line)
```
Comment (rc) [inline sfepy/discrete/fem/meshio.py]: This line can be simply removed. The other readers do not print such information. (Alternative: add `verbose` argument to `read()`.)

**Comment by zitkat** [comment]
> Thanks! Could you rebase on master so that the merge commits disappear? I can help with that, if needed.

I don't see any merge commits, I did rebase just after 34b0110 so maybe it resolved them.

File: sfepy/discrete/fem/meshio.py (hunk 1 above)
Comment (zitkat) [reply sfepy/discrete/fem/meshio.py]: I removed it.

**Comment by rc** [comment]
Yes, I hoped rebase would remove the merge commits, thanks.

**Pull request merged**